src/app/modules/main/communities/tokens/module.nim

```diff
@@ -145,11 +147,12 @@ method deployCollectibles*(self: Module, communityId: string, fromAddress: strin
   self.tempTokenMetadata.tokenType = TokenType.ERC721
   self.tempTokenMetadata.image = singletonInstance.utils.formatImagePath(image)
```

Comment:
This will not be used and can be removed too. Also image field from tempTokenMetadata.